src/components/shared/CustomWidthTooltip.tsx

```diff
@@ -0,0 +1,14 @@
+import { styled, Tooltip, tooltipClasses, TooltipProps } from '@mui/material';
+
+const CustomWidthTooltip = styled(({ className, ...props }: TooltipProps) => (
```

Comment:
Not for this PR

This is interesting... I never really thought about moving the `styled` stuff into stand alone components as they're almost always just needed in one place.

We should take some time and move these to their own folders so it is easy to see which MUI components we are deeply customizing.